CC/Sounder/receiver.cc

```diff
@@ -185,26 +185,44 @@ void Receiver::loopRecv(int tid, int core_id, SampleBuffer* rx_buffer)
     if (bsSdrCh == 2)
         samp_buffer[1] = samp_buffer1.data();
 
+    int cell = 0;
     if (kUseUHD) {
-        std::cout << "Sync BS host and FPGA timestamp..." << std::endl;
-        baseRadioSet_->radioRx(0, 0, samp_buffer.data(), config_->sampsPerSymbol, rxTimeBs);
-        // schedule the first beacon in the future
-        txTimeBs = rxTimeBs + config_->sampsPerSymbol * config_->symbolsPerFrame * BEACON_INTERVAL;
-        baseRadioSet_->radioTx(0, 0, beaconbuff.data(), 2, txTimeBs);
-        long long bsInitRxOffset = txTimeBs - rxTimeBs;
-        for (int it = 0; it < std::floor(bsInitRxOffset / config_->sampsPerSymbol); it++) {
-            baseRadioSet_->radioRx(0, 0, samp_buffer.data(), config_->sampsPerSymbol, rxTimeBs);
+        // For multi-USRP BS perform dummy radioRx to avoid initial late packets
+        int bs_sync_ret = -1;
+        std::cout << "Sync BS host and FPGA timestamp for thread " << tid << "..." << std::endl;
+        for (int it = radio_start; it < radio_end; it++) {
+            // Find cell this USRP belongs to..,
+            for (int i = 0; i <= static_cast<int>(config_->nCells); i++) {
```

Comment:
Can we use `size_t` type for `i`? and `it`?

Reply:
I think it should work, but I simply just copied this part from
https://github.com/renew-wireless/RENEWLab/blob/develop/CC/Sounder/receiver.cc#L212-L218

Should we use `size_t` for in those lines as well?

Reply:
Yes please. I understand there are instances in the code that is not doing this but we are gradually moving toward the Agora code style, with one of the points being to not use `int` in loops etc, unless it is absolutely needed.

Reply:
done! tried to replaced `int` with `size_t` and avoid static cast in a few places